src/hearings/store/actions/hearing-request.action.ts

```diff
@@ -28,8 +29,17 @@ export class SubmitHearingRequest implements Action {
   }
 }
 
+export class NavigateBeginningCancelRequest implements Action {
+  public readonly type = NAVIGATE_BEGINNING_CANCEL_REQUEST;
+
+  constructor(public payload: HearingRequestMainModel) {
+  }
+}
```

Comment:
this hearing request action is used for request hearing only, the cancel shouldn't use ngrx store.